web/src/containers/activity/CostAllocateFFPQuarterly.js

```diff
@@ -67,37 +77,29 @@ class CostAllocateFFPQuarterly extends Component {
           <tbody>
             {['state', 'contractors'].map(name => (
               <Fragment key={name}>
-                <tr
-                  key={name}
-                  className={`align-middle ${
-                    name === 'combined' ? 'bold' : ''
-                  }`}
-                >
-                  <td rowSpan="2" headers={`act_qbudget_ffy${year}`}>
+                <tr key={name}>
+                  <th
+                    rowSpan="2"
+                    scope="row"
+                  >
                     {EXPENSE_NAME_DISPLAY[name]}
-                  </td>
+                  </th>
                   <Fragment key={year}>
                     {QUARTERS.map(q => (
-                      <td
-                        key={q}
-                        className="mono right-align"
-                        headers={`act_qbudget_fy${year} act_qbudget_fy${year}_q${q}`}
-                      >
+                      <td key={q}>
                         <PercentInput
                           name={`ffp-${aKey}-${year}-${q}-${name}`}
                           label={`federal share for ffy ${year}, quarter ${q}, ${name}`}
-                          wrapperClass="m0"
-                          className="m0 input input-condensed mono right-align"
+                          wrapperClass="budget-table--input-holder"
+                          className="budget-table--input__number"
                           onChange={this.handleChange(year, q, name)}
                           value={quarterlyFFP[year][q][name].percent * 100}
                           hideLabel
+                          aria-controls={`ffp-${aKey}-${year}-${q}-${name}-dollar-equivalent`}
```

Comment:
![oooooh](https://media.giphy.com/media/6FymBmqKeBrl6/giphy.gif)

Reply:
run awaaaaaayyyyy

Reply:
Actually, I meant to put a comment in this part... there's some _weird_ interactive shit going on here and I'm really not entirely sure what an accessible workflow should be. I opened a ticket to assess it further. 😬